src/components/Popover/__tests__/Popover.test.tsx

```diff
@@ -0,0 +1,291 @@
+import React from 'react';
+import {fireEvent, render, screen} from '@testing-library/react';
+import user from '@testing-library/user-event';
+
+import '../../../tests/utils/fakeTimers';
+import {Popover} from '../Popover';
+import type {PopoverProps} from '../';
+
+const defaultTooltipContent = 'Tooltip';
+const defaultTriggerText = 'Trigger';
+
+const renderPopover = (props?: Partial<PopoverProps>) => (
```

Comment:
Why?

Reply:
Method returns component, render belongs to `render` method of testing library

Reply:
I understand, but this is a common name for function which returns a markup. What prefix would you suggest?

Reply:
It's not flexible approach, because a custom render function could be added for some cases and I believe that each file should look the same. Also, `render` returns `rerender`, which rerenders component and the renderer can be reused

Reply:
`renderSomething` is a common name for the functions which return `jsx`